mandatory/3-function-output.js

```diff
@@ -2,15 +2,20 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
+// Comment : math.random gives us a random number between 0 and 1. then we times it by 10, we will have a random number between 0 and 10.
```

Comment:
From my findings on Google, I could be wrong though, the getRandomNumber function will generate only floating numbers between 1 - 10

Reply:
math.random() gives you a number between 0 and 1 , if you times it by 10 , then gives you 0 to 10